### PR TITLE
feat: activate Sophia syntax highlighting by using specific pygments …

### PR DESCRIPTION
**Pull request by marc0olo**
…version

image

### CODE REVIEW

**Review by hanssv — DISMISSED** [review]

**Review by uwiger — DISMISSED** [review]

File: .github/workflows/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 mkdocs==1.2.3
 mkdocs-simple-hooks==0.1.3
 mkdocs-material==7.1.9
-mike==1.0.1
\ No newline at end of file
+mike==1.0.1
+git+git://github.com/pygments/pygments@38cd147859fdaeb73a0a2018601f4ec5277798bc#egg=pygments
```
Comment (dincho) [inline .github/workflows/requirements.txt:5]: that's until they release right ?

Comment (marc0olo) [reply .github/workflows/requirements.txt:5]: yes - but last release was in august so I don't expect it anytime soon. I will watch and update to latest release as soon as I observe it

Comment (marc0olo) [reply .github/workflows/requirements.txt:5]: but I just noticed an error in the PR

Comment (marc0olo) [reply .github/workflows/requirements.txt:5]: fixed 😅 .... just to make sure the lib is actually used with the specific commit version

**Review by dincho — APPROVED** [review]

**Review by hanssv — APPROVED** [review]

**Review by ghallak — APPROVED** [review]

**Pull request merged**